lnwallet/channel.go

```diff
@@ -4622,6 +4622,17 @@ func (lc *LightningChannel) SignNextCommitment() (*NewCommitState, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// We'll need to send over the signatures to the remote party in the
+	// order as they appear on the commitment transaction after BIP 69
+	// sorting.
+	slices.SortFunc(sigBatch, func(i, j SignJob) int {
```

Comment:
This change isn't necessarily _wrong_, but it shouldn't really be necessary either. Since the order is only important for the on-chain transaction. What was the reason you moved this up again?
Trying to find out if I should take over this commit into [PART3](https://github.com/lightningnetwork/lnd/pull/9072) or not...